user-pool-commits.md

```diff
@@ -0,0 +1,66 @@
+# Make git commits from a user pool
+
+## Description
+
+- Makes things a little bit faster if you were to commit as multiple users to a single repository
+
+## Prerequisites
+
+- User should have git installed on their machine
+
+## Steps to Setup
+
+1. **Add the following bash script to the folder containing cloned repo. Recommended file name is 'cfup.sh':**
+
+```bash
+#!/bin/bash
+
```

Comment:
let's add some error handling here to check weather user have git installed.. 
```
if ! command -v git &>/dev/null; then
    echo "git is not installed. Please install git to use this script."
    exit 1
fi
```
what do you think?

Reply:
Someone without git installed won't be looking in GitHub for gists. But we'll add this just in case.